Assets/Scripts/Game.cs

```diff
@@ -72,6 +75,13 @@ void Start()
         // Create some things
         uiScaler = new UIScaler(uICanvas);
         config = new ConfigFile();
+        currentLang = 1; // English
+
+        if (config.data.Get("MoMConfig") != null)
```

Comment:
I get that D2E doesn't have translations now, but I don't see why support shouldn't be common.  The format is the same and is still has this:

```
KEY,English,Spanish,French,German,Italian,Portuguese,Polish,Japanese,Chinese,Czech
```

If we went to exclude language options because we know they aren't actually available we could either check a test key (ie: CAMPAIGN_1_TITLE) or maybe even check all keys to see if that language has anything.

Either way I don't particularly like language selection being under "MoMConfig", even if for now we don't implemented it for any other game type.

tldr; please move language settings out of 'MoMConfig', but you can leave the rest if you want.

As for the extra files, even better would be to put them in a sensible place!  I have fixed this in development.